src/Event/UserUuid.php

```diff
@@ -0,0 +1,31 @@
+<?php
+declare(strict_types=1);
+
+namespace Landingi\EventStoreBundle\Event;
+
+use Symfony\Component\Uid\Uuid;
+
+final class UserUuid implements \JsonSerializable
+{
+    private Uuid $uuid;
+
+    public function __construct(Uuid $uuid)
+    {
+        $this->uuid = $uuid;
+    }
+
+    public function getUuid(): Uuid
+    {
+        return $this->uuid;
+    }
+
+    public function jsonSerialize(): string
+    {
+        return $this->uuid->jsonSerialize();
```

Comment:
Odwróciłbym to. W takim wypadku zrobiłbym tutaj `(string) $this->uuid`, bo jak będziesz mieć przykładowo coś co ma 2 wartości, to z założenia `jsonSerialize` musi zwrócić tablicę, a `__toString` łańcuch znaków. Wtedy byś wywołał w metodzie`__toString` `return json_encode($this->uuid)`. Tak mnie się wydaje, ale mogę się mylić :)

Reply:
Niestety to musi tak być, ponieważ każdy z VO Eventu ma implementacje jsonSerialize, a do `UserUuid` przekazywany jest Symfony\Uid, które aby zrzutować do stringa trzeba użyć `jsonSerialize()`